Sources/ServerSyncJob.swift

```diff
@@ -38,6 +38,10 @@ extension ServerSyncJob: Codable {
         case openEventTypeKey
         case deliveryEventTypeKey
     }
+    
+    private enum ServerSyncJobError: Error {
+        case ParseError(reason: String)
```

Comment:
`Error` protocol in Swift has a property called `localizedDescription` which is supposed to provide a description/reason for the given error. It would be a good idea to override this property here.

Reply:
👍 thank you!